server/routes/api/profile/profileDBHandler.ts

```diff
@@ -0,0 +1,27 @@
+import db from "../../../utils/db/db";
+import { END_TIMESTAMP } from "../../../utils/global/globals";
+
+import { ParentProfile,BabysitterProfile } from "./profileTypes";
+
+export default class DBHandler {
+    async getBabySitterProfile(babysitterId: number): Promise<BabysitterProfile[]> {
+        const babysitterQuery = `SELECT * 
```

Comment:
Write exactly what columns you want from the select * and you need to format it to fit the type you are returning.
For example - phone_number to phoneNumber